layouts/partials/install.html

```diff
@@ -2,114 +2,108 @@
+              <h5>For servers or big data</h5>
               <p>
-                Using IPFS for servers or large datasets? Cluster automatically allocates, replicates, and tracks your
-                data as pinsets across multiple IPFS nodes.
+                Automatically allocate, replicate, and track your data as pinsets across multiple IPFS nodes.
               </p>
+              <div><a class="button button-outlined" href="https://cluster.ipfs.io/" target="_blank" rel="noopener noreferrer">Get Cluster</a></div>
             </div>
           </div>
-        </a>
       </div>
+    </div>
+
+    <h3 class="content-center" style="margin-top: 70px;">Build with IPFS</h3>
+    <div class="grid-flex-container">
 
       <div class="grid-flex-cell-1of2">
-        <a href="https://github.com/ipfs/go-ipfs" target="_blank">
           <div class="install-item">
             <div class="install-item-image">
-              <img src="/images/go-ipfs-hex-logo.png" />
+              <a href="https://github.com/ipfs/go-ipfs" target="_blank" rel="noopener noreferrer"><img src="/images/go-ipfs-hex-logo.png" /></a>
             </div>
             <div class="install-description">
-              <h4>Go implementation</h4>
+              <a href="https://github.com/ipfs/go-ipfs" target="_blank" rel="noopener noreferrer"><h4>Go implementation</h4></a>
               <p>
-                If you build on Go, then the main IPFS implementation is for you. Includes core implementation, daemon server, CLI tooling, and more.
+                The original IPFS, with core implementation, daemon server, CLI tooling, and more.
               </p>
+              <div><a class="button button-outlined" href="https://github.com/ipfs/go-ipfs" target="_blank" rel="noopener noreferrer">Get go-ipfs</a></div>
             </div>
           </div>
-        </a>
       </div>
 
       <div class="grid-flex-cell-1of2">
-        <a href="https://js.ipfs.io" target="_blank">
           <div class="install-item">
             <div class="install-item-image">
-              <img src="/images/js-ipfs-logo.png" />
+              <a href="https://js.ipfs.io" target="_blank" rel="noopener noreferrer"><img src="/images/js-ipfs-logo.png" /></a>
             </div>
             <div class="install-description">
-              <h4>JavaScript implementation</h4>
+              <a href="https://js.ipfs.io" target="_blank" rel="noopener noreferrer"><h4>JS implementation</h4></a>
               <p>
-                Written entirely in JavaScript, js-ipfs paves the way for the browser implementation of IPFS and opens
-                the
-                door to a world of possibilities.
+                Written entirely in JavaScript for a world of possibilities in browser implementations.
               </p>
+              <div><a class="button button-outlined" href="https://js.ipfs.io" target="_blank" rel="noopener noreferrer">Get js-ipfs</a></div>
```

Comment:
For the backlog: I wonder if we want this to instead point js-ipfs on NPM and have the NPM logo here somehow. I wonder what the cost or benefit of redirecting to a whole other website is, vs pointing to how JS developers install software. Could still include a "learn more at js.ipfs.io" or something like that.

Reply:
Yep. I'd really like to rework this whole section a little harder in future to relink directly to artifacts whenever appropriate -- like to OS-correct executables for Desktop, browser-correct store links for Companion, single-line install commands for package managers of choice, etc. It's on the radar but I didn't want to upset the apple cart in this iteration.